examples/fluids/index.md

```diff
@@ -301,6 +301,18 @@ Note that this wave speed is specific to ideal gases as $\gamma$ is an ideal gas
 Currently, this demo provides three types of problems/physical models that can be selected at run time via the option `-problem`.
 {ref}`problem-advection`, the problem of the transport of energy in a uniform vector velocity field, {ref}`problem-euler-vortex`, the exact solution to the Euler equations, and the so called {ref}`problem-density-current` problem.
 
+### Subgrid Stress Modeling
+
+When a fluid simulation is under-resolved (the smallest length scale resolved by the grid is much larger than the Kolmogorov length scale), this is mathematically interpreted as filtering the Navier-Stokes equations. 
+This filtering operation results in an extra stress-like term in the equations, representing the effect of unresolved (or "subgrid" scale) structures in the flow. 
+To close the problem, this term is modeled using a subgrid stress (SGS) model.
+
+#### Data-driven SGS Model
+
+The data-driven SGS model implemented here uses a small neural network (1 layer of 20 neurons) to compute the SGS term.
+The SGS tensor is calculated at nodes using an $L^2$ projection of the velocity gradient and grid anisotropy tensor, and then interpolated onto quadrature points.
+Parameters for the neural network are put into files in a directory found in `-sgs_model_dd_parameter_dir`.
```

Comment:
I take it this is multiple files in the directory?

Reply:
Yes. I was planning on mentioning the format and files required explicitly.

Reply:
Done